dipy/core/gradients.py

```diff
@@ -127,6 +194,25 @@ def gradient_table_from_bvals_bvecs(bvals, bvecs, b0_threshold=50, atol=1e-2,
     atol : float
         Each vector in `bvecs` must be a unit vectors up to a tolerance of
         `atol`.
+    btens : can be any of three options
+        1. a string specifying the shape of the encoding tensor for all volumes
+           in data. Options: 'LTE', 'PTE', 'STE' corresponding to linear,
```

Comment:
Use the enum instead of the strings?

Reply:
Let's leave this for another pull request after we have agreed on an optimal way of doing it.